docs/note.md

```diff
@@ -493,3 +493,32 @@ ffmpeg -ss 00:00:05 -i ${vdirname}/${vfname}mp4 -frames:v 1 ${vdirname}/${vfname
 {{< alert title="Note" color="note" >}}
 The `2gif` commands only turn the first 5 seconds of a video into a low res gif.
 {{< /alert >}}
+
+## Images
+
+**Place images in the `assets` folder.**
+
+While you can include images using markdown syntax (`![ALT text](file)`), please use the following shortcode (without the slashes) instead:
+
+```md
+\{\{<imgproc src="/installation/thumbnails/raspberry-pi-4-b-2gb.png" resize="x60" declaredimensions=true alt="Raspberry Pi">\}\}
+
+\{\{<imgproc src="/installation/thumbnails/raspberry-pi-4-b-2gb.png" resize="x200" declaredimensions=true alt="Raspberry Pi">\}\}
+```
+
+{{<imgproc src="/installation/thumbnails/raspberry-pi-4-b-2gb.png" resize="x60" declaredimensions=true alt="Raspberry Pi">}}
+
+{{<imgproc src="/installation/thumbnails/raspberry-pi-4-b-2gb.png" resize="x200" declaredimensions=true alt="Raspberry Pi">}}
+
+
+The `imgproc` shortcode will:
+- convert the image into the `webp` format which is more efficient and resize the image
+- resize the image in the current format and set that image as a backup in case `webp` is not supported
+
+For more information on the resize options see [Image Processing](https://gohugo.io/content-management/image-processing/).
+
+{{< alert title="Important" color="note" >}}
+Only specify `declaredimensions` if the image is **not** responsive, that means if it doesn't resize.
+
+If it does resize, use the largest size the image can take up as the image to `resize` the image to.
```

Comment:
No it's more if it's in a card and needs to resize then you can't use this property

Reply:
Couldn't any image resize if you wanted it to? Or I guess, what is an example of an image that doesn't resize and that couldn't resize?

Reply:
On the front page the small board icons should never be a different size than they are.
The pictures in cards, however, need to resize because they change size based on the available screen space.
Screenshot images are as big as they can be generally but on mobile they're smaller. 

So basically the only images that you'd want to use `declaredimensions` on are the ones that take up the same space on mobile as on desktop. 

I'll add this explanation to the docs. It's also entirely possible that I will change in the future how cards work so that we can declare the dimensions in the html. There is an issue with content shifting around as it is loaded that isn't ideal for web experience reasons. But for now this will do.